build/unix/modulemap.overlay.yaml.in.install

```diff
@@ -0,0 +1,19 @@
+{
+  'version': 0,
+  'roots': [
+    { 'name': '@__libcpp_full_path@', 'type': 'directory',
+      'contents': [
+        { 'name': 'module.modulemap', 'type': 'file',
+          'external-contents': '@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_INCLUDEDIR@/stl.modulemap'
+        }
+      ]
+    },
+    { 'name': '/usr/include/', 'type': 'directory',
```

Comment:
Is this going to work on Mac, that has no `/usr/include/` anymore? Or on other systems that get headers from elsewhere? I know this was there before... Just asking in case it makes sense to change this to something else.

Reply:
I guess it doesn't work, but I'm not sure for the alternative (this is the way how it was used in the previous modulemap.overlay.yaml) And also not sure if we should even think about Mac. @Teemperor @vgvassilev any idea?

Reply:
We don't use the modulemap overlay on macOS. Well, technically we don't use it on libc++ installations which is the default for macOS. The code path for this is a bit obfuscated...

Reply:
> Well, technically we don't use it on libc++ installations which is the default for macOS

That's strange, considering the first entry has `'name': '@__libcpp_full_path@'` in it...

Reply:
I shall correct my statement: "The code path for this is ~~a bit~~ very obfuscated..."

But yes, we should rename this after we landed this PR.